README.md

```diff
@@ -1,6 +1,6 @@
 # Money
 
-Money is a small PHP library that helps you format numbers to currencies, including INR (Indian Rupee). It's a simple alternative to money_format().
+Money is a small PHP library that helps you format numbers to currencies, including NGN (Nigrian Naira). It's a simple alternative to money_format().
```

Comment:
Thanks for the PR!

One quick suggestion, let's keep INR in the sentence and add NGN:
```suggestion
Money is a small PHP library that helps you format numbers to currencies, including INR (Indian Rupee) and NGN (Nigrian Naira). It's a simple alternative to money_format().
```
It would be great if you could add NGN to the [supported currencies section](https://github.com/gerardojbaez/money/blob/5de9058ff396622a041999ef28876da77f2352bb/README.md#currencies-supported).

One final note, you wrote "Nigrian", did you meant "Nigerian"?